cmd/convox/templates/init/rails/docker-compose.yml

```diff
@@ -6,3 +6,5 @@ web:
   ports:
     - 80:4000
     - 443:4001
+  volumes:
```

Comment:
There are a plethora of other things that might mutate /app and get blown away. Assets precompile?